src/css/_grid.scss

```diff
@@ -50,3 +51,8 @@ $container-max-widths: (
 @import "../../node_modules/bootstrap/scss/grid";
 @import "../../node_modules/bootstrap/scss/utilities/display";
 @import "../../node_modules/bootstrap/scss/utilities/flex";
+
+.app-container {
+  background-color: $ams-wit;
+  max-width: 1400px;
```

Comment:
Shouldn't this value be kept in a variable and be kept in breakpoints? Or at the top of this scss?
It feels like more or less dependent on the breakpoints. If any breakpoint becomes larger than 1400px, who notices that this value needs to be updated?